lms/djangoapps/teams/tests/test_views.py

```diff
@@ -0,0 +1,445 @@
+"""Tests for the teams API at the HTTP request level."""
+# pylint: disable=maybe-no-member
+import json
+
+import ddt
+
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment
+from xmodule.modulestore.tests.factories import CourseFactory
+from .factories import CourseTeamFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+
+@ddt.ddt
+class TestTeamAPI(APITestCase, ModuleStoreTestCase):
+    """
+    Tests of the Team API
+    """
+
+    test_password = 'password'
+
+    topics_count = 10
+
+    def setUp(self):
+        super(TestTeamAPI, self).setUp()
+
+        teams_configuration = {
+            'topics':
+            [
+                {
+                    'id': 'topic_{}'.format(i),
+                    'name': 'Topic {}'.format(i),
```

Comment:
Looking at make_topic in /edx-platform/common/lib/xmodule/xmodule/tests/test_course_module.py, the expected fields are "id", "display_name", and "description".